### ISSUE TITLE
improper `checkout list` display, when any tag is checked out (walkback in 3.0.56)

### ISSUE THREAD

**Issue by dalehenrich** [GemTalk/Jadeite#311]
This is what the git output of the `git branch` command looks like (from the gem log):
```
'''set -e; cd $ROWAN_PROJECTS_HOME/Rowan; git --git-dir $ROWAN_PROJECTS_HOME/Rowan/.git --work-tree $ROWAN_PROJECTS_HOME/Rowan branch '''
'* (HEAD detached at v0.1.0-alpha)
  candidate
  dale
  issue_141
  issue_141_test_0
  issue_141_test_1
  master
'
```
and this is what the list of branches looks like in Jadeite:
![image](https://user-images.githubusercontent.com/623951/40209157-9bfe5994-59f2-11e8-8a88-8d4736153405.png)
Not a priority at this point in time ...

**Comment by dalehenrich** [comment]
walkback attempting to reproduce this bug in 3.0.56:
```
a MessageNotUnderstood occurred (error 2010), a Character does not understand  #'command:'
MessageNotUnderstood >> defaultAction @2 line 3
MessageNotUnderstood (AbstractException) >> _signalWith: @5 line 25
MessageNotUnderstood (AbstractException) >> signal @2 line 47
Character (Object) >> doesNotUnderstand: @9 line 10
Character (Object) >> _doesNotUnderstand:args:envId:reason: @7 line 12
RowanCommandResult class >> addResult: @2 line 2
[] in  ExecBlock1 (RowanQueryService) >> returnQueryToClient @2 line 4
String (Collection) >> do: @5 line 10
RowanQueryService >> returnQueryToClient @2 line 3
RowanQueryService >> projectBranches: @8 line 6
RowanQueryService (RowanService) >> servicePerform:withArguments: @2 line 2
[] in  ExecBlock1 (JadeServer) >> updateFromSton: @10 line 12
Array (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> updateFromSton: @7 line 6
GsNMethod class >> _gsReturnToC @1 line 1
```

**Comment by ericwinger** [comment]
I have a fix in 3.0.57 for the poor display when HEAD is checked out. 

As far as the walkback, ``projectBranches:`` no longer sends ``returnQueryToClient`` so I suspect an older server was being used during the previous walkback. Here's the correct version. 
```
projectBranches: projectName

	| project  |
	project := (RwProject newNamed: projectName). 
	queryResults := Rowan gitTools gitbranchIn: project repositoryRootPath with: ''.
	RowanCommandResult addResult: self
```
